src/perftest_parameters.c

```diff
@@ -2819,7 +2825,11 @@ int parser(struct perftest_parameters *user_param,char *argv[], int argc)
 					use_hl_flag = 0;
 				}
 				if (flow_label_flag) {
-					CHECK_VALUE_NON_NEGATIVE(user_param->flow_label,int,"flow label",not_int_ptr);
+					CHECK_VALUE(user_param->flow_label,int,"flow label",not_int_ptr);
+					if (user_param->connection_type == RawEth && user_param->flow_label < 0) {
+						fprintf(stderr," flow label must be non-negative for RawEth\n");
+						return 1;
```

Comment:
add free(duplicates_checker) and lets replace "return 1" by enum "return FAILURE"

Reply:
@HassanKhadour 
What's the meaning of ```add free(duplicates_checker)```?
The original code checks that ```flow_label``` isn't negative value when ```connection_type is RawEth```. So, I kept the original code logic there.

Reply:
I know, this part of code (long options) needs enhancements, planning to do it in this period, so nvm, I'll fix in a latter commit.